Project-3/README.md

```diff
@@ -11,67 +11,466 @@ For any question involving the use of Protege, please be sure to import:
+
 ```
 
-5. Using BFO and RO, model the following scenario: the content of an rdf file is represented in two serializations - one in Turtle, one in XML - which are sent from one computer to two distinct computers on the same network.   
+5. Using BFO and RO, model the following scenario: the content of an rdf file is represented 
+in two serializations - one in Turtle, one in XML - which are sent from one computer to two 
+distinct computers on the same network.   
+
+RDF File Content, Turtle and XML are all generically dependent continuants. 
+RDF File Content has_model in Turtle and XML. And they are all carried by Computer.
+Computer 1 enables the process of Data Transmission, which ends_with another process, 
+Data Reception, which has computers 2 and 3 as participants. So this part is intended to show
+that there is some data (Turtle and XML) which are sent from Computers 1 to Computer 2 and 3.
+Finally, computers 1, 2, and 3 are all parts_of network I,
+which is instance_of Object Aggregate.)
+
+![image](https://user-images.githubusercontent.com/117365623/224577326-7b77203b-f198-4678-95d5-240c53cff36a.png)
 
 
 6. Using Protege, place these in the BFO hierarchy where you think they fit best:
 ```
+  Some of these questions are of course ambigous. In a real-life situation, we would talk with 
+  the domain experts/ client who asked us to build the ontology and understand what of the 
+  options they are meaning.
+
   (a) Bach's Well-Tempered Clavier
+  Generically dependent continuant, or a process (the execution), or even an object (the sheets)
   (b) Chair of the UB Philosophy Department
-  (c) SARS-CoV-2
+  Role
+  (c) SARS-COV-2
+  Object (apparently we use this term to refer to the virus, not the disease/ disorder)
   (d) Mexico City
+  Object aggregate, but could also be a two- or three-dimensional spatial region 
+  (the region where the city is located) or a site (the empty space where people roam)
   (e) The trunk of a minivan
+  Site (the empty space inside the trunk). Or an object/ object aggregate.
   (f) Occupation
+  Role
   (g) Ocean
+  Object aggregate, but could also be a two- or three-dimensional spatial region (the region the 
+  ocean occupies) or a site (the empty space where water goes).
   (h) Lake
-```
+  Object aggregate, but could also be a two- or three-dimensional spatial region (the region the 
+  Lake occupies) or a site (the empty space where water goes).
+  ```
 
 7. True or False; explain your answers:
 ```
   (a) An instance of Material Entity can have an instance of Immaterial Entity as part.
+  True (see BS book pag. 108)
   (b) An instance of Immaterial Entity can have an instance of Material Entity as part.
+  False by def. and by some properties in the ontology.
   (c) An organization may have another organization as part.
+  True. A committee can have a subcommittee as a part. 
   (d) An organization may have no members as part. 
+  If organizations are object aggregates, this is false.
```

Comment:
The current specification allows that organizations may have no members at some point, but they must've had members when created.